solc/CommandLineInterface.h

```diff
@@ -79,7 +79,6 @@ class CommandLineInterface
 	void handleABI(std::string const& _contract);
 	void handleNatspec(bool _natspecDev, std::string const& _contract);
 	void handleGasEstimation(std::string const& _contract);
-	void handleFormal();
```

Comment:
This was deprecated long time ago.

Reply:
Yeah, looks like there's no definition and no references to it anywhere.